src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Simplification/Simplifiers/CastSimplifier.cs

```diff
@@ -2,16 +2,12 @@
 // The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
looking at this diff is pointless.  it is a totally new impl.  it will only make reading each commit one at a time.

Reply:
or i recommend just looking at the final form.